multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -66,6 +66,13 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # IP Address for controller in Kubernetes subnet
 # controllerIP: 10.0.0.50
 
+# Comma seperated list of Etcd endpoints. By default this needs to be set to "http://<controllerIP>:2379" 
+# if custom etcdEndpoints are specified then a security group id can also be specified which will expose
+# the etcd security group to the worker & controller instances.
+# etcdEndpoints: "http://10.0.0.50:2379"
+# etcdSecurityGroupId: ""
```

Comment:
@cgag because localhost would prevent the other k8s nodes from being able to reach the etcd endpoint. 

The first endpoint from the EtcdEndpoints list is used to seed both the worker and controller cloud-config.